src/components/Todos/model.js

```diff
@@ -27,6 +30,10 @@ function searchTodoIndex(todosList, todoid) {
   return null;
 }
 
+// MAKE MODIFICATION STREAM
+// A function that takes the intent on the todo list
```

Comment:
intent is the function. actions are the output of the intent. This function here takes actions, not intent.